DbgProvider/DbgProvider.csproj

```diff
@@ -456,15 +457,15 @@
   <PropertyGroup>
     <FileDescription>Core DbgShell binary: implements the PowerShell provider and C# object model.</FileDescription>
   </PropertyGroup>
-  <Import Project="..\packages\Unofficial.Microsoft.TextTemplating.BuildTasks.15.0.0\build\Unofficial.Microsoft.TextTemplating.BuildTasks.targets" Condition="Exists('..\packages\Unofficial.Microsoft.TextTemplating.BuildTasks.15.0.0\build\Unofficial.Microsoft.TextTemplating.BuildTasks.targets')" />
   <Import Project="..\packages\AddGitVersionInfo.1.0.0.1\build\AddGitVersionInfo.targets" Condition="Exists('..\packages\AddGitVersionInfo.1.0.0.1\build\AddGitVersionInfo.targets')" />
   <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
     <PropertyGroup>
       <ErrorText>This project references NuGet package(s) that are missing on this computer. Enable NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
     </PropertyGroup>
     <Error Condition="!Exists('..\packages\AddGitVersionInfo.1.0.0.1\build\AddGitVersionInfo.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\AddGitVersionInfo.1.0.0.1\build\AddGitVersionInfo.targets'))" />
-    <Error Condition="!Exists('..\packages\Unofficial.Microsoft.TextTemplating.BuildTasks.15.0.0\build\Unofficial.Microsoft.TextTemplating.BuildTasks.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\Unofficial.Microsoft.TextTemplating.BuildTasks.15.0.0\build\Unofficial.Microsoft.TextTemplating.BuildTasks.targets'))" />
+    <Error Condition="!Exists('..\packages\Unofficial.Microsoft.TextTemplating.BuildTasks.16.0.0.1\build\Unofficial.Microsoft.TextTemplating.BuildTasks.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\Unofficial.Microsoft.TextTemplating.BuildTasks.16.0.0.1\build\Unofficial.Microsoft.TextTemplating.BuildTasks.targets'))" />
   </Target>
+  <Import Project="..\packages\Unofficial.Microsoft.TextTemplating.BuildTasks.16.0.0.1\build\Unofficial.Microsoft.TextTemplating.BuildTasks.targets" Condition="Exists('..\packages\Unofficial.Microsoft.TextTemplating.BuildTasks.16.0.0.1\build\Unofficial.Microsoft.TextTemplating.BuildTasks.targets')" />
```

Comment:
I also made this change (to pick up the latest T4 build tasks), and I just stuffed it into master, along with some other changes to upgrade to the VC142 toolset. I also upgraded the appveyor build to use the VS2019 image. Strangely, a couple of the tests still fail. I had seen these failures locally, and addressed them with [19576eda44d3a92c2](https://github.com/microsoft/DbgShell/commit/19576eda44d3a92c20ed5bbe6674dd1f269df621), but they still fail on the build server. Once you've picked up my changes, do those tests still fail for you locally?